# Relatório de Análise 1

## Importando arquivos 

In [2]:
import pandas as pd

In [14]:
#Utilizamos 'sep' para indicar o parametro que o dataset está dividido

dataset = pd.read_csv('aluguel.csv', sep = ';')

# Analisando os tipos de dados

In [4]:
#Chamando as primeiras linhas do dataset com 'head'
dataset.head()

Tipo           Bairro  Quartos  Vagas  Suites  Area  \
0                Quitinete       Copacabana        1      0       0    40   
1                     Casa  Jardim Botânico        2      0       1   100   
2  Conjunto Comercial/Sala  Barra da Tijuca        0      4       0   150   
3              Apartamento           Centro        1      0       0    15   
4              Apartamento     Higienópolis        1      0       0    48   

    Valor  Condominio    IPTU  
0  1700.0       500.0    60.0  
1  7000.0         NaN     NaN  
2  5200.0      4020.0  1111.0  
3   800.0       390.0    20.0  
4   800.0       230.0     NaN

In [5]:
# 'Info' mostra todas as informações sobre o dataset 
dataset.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 32960 entries, 0 to 32959
Data columns (total 9 columns):
 #   Column      Non-Null Count  Dtype  
---  ------      --------------  -----  
 0   Tipo        32960 non-null  object 
 1   Bairro      32960 non-null  object 
 2   Quartos     32960 non-null  int64  
 3   Vagas       32960 non-null  int64  
 4   Suites      32960 non-null  int64  
 5   Area        32960 non-null  int64  
 6   Valor       32943 non-null  float64
 7   Condominio  28867 non-null  float64
 8   IPTU        22723 non-null  float64
dtypes: float64(3), int64(4), object(2)
memory usage: 2.3+ MB


#### Conclusões baseadas no 'info()'
- Podemos observar que o dataset contém **32960** linhas e **9** colunas
---
- A coluna 'Valor' tem somente **32943** linhas. Ou seja, há valores nulos na coluna. A partir disso devemos analisar se estes valores nulos vão afetar nossas análises futuras. O mesmo serve para a coluna 'Condominio' e 'IPTU'
- Condominio: **28867** linhas
- IPTU: **22723** linhas

# Informações gerais sobre a base de dados

In [7]:
dataset.dtypes

Tipo           object
Bairro         object
Quartos         int64
Vagas           int64
Suites          int64
Area            int64
Valor         float64
Condominio    float64
IPTU          float64
dtype: object

In [16]:
# Criando um DataFrame sobre os tipos de dados para melhor visualização e atribuindo uma coluna com nome 'Tipo de Dados'
tipo_de_dados = pd.DataFrame(dataset.dtypes, columns = ['Tipo de Dados'])
tipo_de_dados

Tipo de Dados
Tipo              object
Bairro            object
Quartos            int64
Vagas              int64
Suites             int64
Area               int64
Valor            float64
Condominio       float64
IPTU             float64

In [17]:
# Atribuindo nome a primeira coluna 
tipo_de_dados.columns.name = 'Variaveis'
tipo_de_dados

Variaveis  Tipo de Dados
Tipo              object
Bairro            object
Quartos            int64
Vagas              int64
Suites             int64
Area               int64
Valor            float64
Condominio       float64
IPTU             float64

In [18]:
#'shape' mostra a quantidade de linhas e colunas 
dataset.shape

(32960, 9)